functions/src/api/donations/donations.ts

```diff
@@ -15,4 +15,33 @@ router.post('/', async (req: any, res: any) => {
     res.json(donation);
 });
 
+router.get('/:id', async (req: any, res: any) => {
+    const donationId = req.params.id;
+    const donation = await get(DONATIONS_COLLECTION, donationId);
+    const lineItemIds = donation.line_item_ids;
```

Comment:
I don't think line_item_ids is a field on the donation object. All of the donation data is stored together and the line items are added to their own collection after. Just returning donation should be enough here, same for the all donations endpoint.